server/src/main/xsd/gateway-config-201503.xsd

```diff
@@ -0,0 +1,1288 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is a new namespace required?  It seems all old namespace configs would parse in this one.  

If we do update the namespace does this affect the next version we release the community gateway at.